packages/dashboard/src/pages/SignUp/steps/ConnectSocialMedia.tsx

```diff
@@ -50,20 +50,22 @@ export const ConnectSocialMedia = () => {
                             {t.go_back()}
                         </Button>
                     </Stack>
-                    {definedSocialNetworks.map(({ networkIdentifier }) => (
-                        <ActionCard
-                            key={networkIdentifier}
-                            title={t.create_account_connect_social_media({
-                                type: upperFirst(networkIdentifier),
-                            })}
-                            icon={SOCIAL_MEDIA_ICON_MAPPING[networkIdentifier]}
-                            action={{
-                                type: 'primary',
-                                text: t.connect(),
-                                handler: () => handleConnect(networkIdentifier),
-                            }}
-                        />
-                    ))}
+                    {definedSocialNetworks
+                        .filter((x) => x.networkIdentifier !== EnhanceableSite.Mirror)
```

Comment:
you should add a new property called something like `hideInSignUp` and filter by this property instead of doing this.

Reply:
Ok, I will refactor this.